samples/msal-browser-samples/vue3-sample-app/package.json

```diff
@@ -4,18 +4,21 @@
   "scripts": {
     "start": "vite",
     "build": "vue-tsc --noEmit && vite build",
-    "serve": "vite preview"
+    "serve": "vite preview",
+    "build:package": "cd ../../../lib/msal-browser && npm run build:all",
+    "install:local": "npm install ../../../lib/msal-browser",
+    "install:published": "npm install @azure/msal-browser@latest"
   },
   "dependencies": {
-    "@azure/msal-browser": "^2.26.0",
-    "element-plus": "^2.2.6",
-    "vue": "^3.2.37",
-    "vue-router": "^4.0.16"
+    "@azure/msal-browser": "file:../../../lib/msal-browser",
```

Comment:
Let's keep the published version here since we dont have tests for this or develop against this.

Reply:
sure thing -done